chain/network-primitives/src/network_protocol/edge.rs

```diff
@@ -4,6 +4,13 @@ use near_primitives::borsh::maybestd::sync::Arc;
 use near_primitives::hash::CryptoHash;
 use near_primitives::network::PeerId;
 
+use crate::time::Utc;
+
+// We'd treat all nonces that are below this values as 'old style' (without any expiration time).
+// And all nonces above this value as new style (that would expire after some time).
+// This value is also couple days before the FakeClock UTC start (to make it easier in unittests).
+pub const EDGE_MIN_TIMESTAMP_NONCE: u64 = 88108233;
```

Comment:
Please select a reasonable timestamp regardless of what FakeClock does.
Annotate in the comment what is the actual date that this timestamp represents, since this value affects the semantics of the prod code.
Consider making this constant strongly typed by using Utc::from_unix_timestamp (Result::unwrap is not const, so this will need to be wrapped in a Lazy, I guess).

You can adjust the FakeClock default initial timestamp accordingly (perhaps even force it to be EDGE_MIN_TIMESTAMP_NONCE + some duration, and add a comment that this is important so that tests are executed in environment similar to prod), and use 
FakeClock::new(initial_utc) explicitly for tests which actually depend on relation wrt this timestamp.

Reply:
ok - updated to the more recent time (and change the test that we cared about.)